src/solver/optimisation/opt_appel_solveur_quadratique.cpp

```diff
@@ -160,7 +172,22 @@ bool OPT_AppelDuSolveurQuadratique(PROBLEME_ANTARES_A_RESOUDRE* ProblemeAResoudr
             }
         }
 #endif
+    }
+}
 
-        return false;
+bool OPT_AppelDuSolveurQuadratique(const Solver::Optimization::OptimizationOptions& options,
+                                   PROBLEME_ANTARES_A_RESOUDRE* ProblemeAResoudre)
+{
+    // as long as sirius quadratic optimization is not supported through or-tools, we have to keep
+    // this code separate
+    if (options.quadraticSolver.compare("sirius") == 0)
+    {
```

Comment:
I find `bool std::string::operator=(const std::string&)` much more readable than `int std::string::compare(const std::string&)`
```suggestion
    if (options.quadraticSolver == "sirius")
    {
```